setup.cfg

```diff
@@ -34,3 +34,22 @@ ignore_missing_imports = True
 [flake8]
 max-line-length = 200
 exclude = venv/*,var/*
+
+# E203 - White space before ':', conflicting with black
+# E712 - comparison to False should be 'if cond is False:' or 'if not cond:', doesn't apply to our code
+# E731 - do not assign a lambda expression, use a def, doesn't apply to our code
```

Comment:
Can you omit `doesn't apply to our code`?

Reply:
done